man/systemd-run.xml

```diff
@@ -221,12 +221,31 @@
 
         <listitem><para>When invoking the command, the transient service connects its standard input and output to the
         terminal <command>systemd-run</command> is invoked on, via a pseudo TTY device. This allows running binaries
-        that expect interactive user input as services, such as interactive command shells.</para>
+        that expect interactive user input/input as services, such as interactive command shells.</para>
 
         <para>Note that
         <citerefentry><refentrytitle>machinectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>'s
         <command>shell</command> command is usually a better alternative for requesting a new, interactive login
-        session on the local host or a local container.</para></listitem>
+        session on the local host or a local container.</para>
+
+        <para>See below for details on how this switch combines with <option>--pipe</option>.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--pipe</option></term>
+        <term><option>-P</option></term>
+
+        <listitem><para>When invoking the command, the transient service's standard input, standard output and standard
+        error are directly connected to the standard inpout, standard output and standard error the
```

Comment:
Typo in "inpout".

Maybe it's easier to say "Standard input, output, and error of the transient service are inherited from the `systemd-run` command itself.". That's what is e.g. used in Python subprocess.Popen docs, and it should be pretty clear for most readers.